tabpy-tools/tabpy_tools/rest.py

```diff
@@ -239,7 +238,7 @@ def __init__(self, name, bases, dict):
                 self.__rest__.add(k)
 
 
-class RESTObject(with_metaclass(_RESTMetaclass, _MutableMapping)):
+class RESTObject(_MutableMapping, metaclass=_RESTMetaclass):
```

Comment:
The intention here is very difficult to reason about, but that's an artifact of existing code so doing minimum here to remove third party dependency